### PR TITLE
WFLY-5222 test for grant custom permission

### PR DESCRIPTION
**Pull request by hsvabek**
WFLY-5222 tests for grant custom permission

https://issues.jboss.org/browse/WFLY-5222

### CODE REVIEW

**Comment by hsvabek** [comment]
retest this please

File: testsuite/integration/secman/src/test/java/org/jboss/as/testsuite/integration/secman/custompermissions/AbstractCustomPermissionServerSetup.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            operation.get("value").set(Arrays.asList(customPermission));
+
+            ManagementOperations.executeOperation(managementClient.getControllerClient(), operation);
+            ServerReload.executeReloadAndWaitForCompletion(managementClient.getControllerClient());
+        }
+    }
+
+    @Override
+    public void tearDown(ManagementClient managementClient, String containerId) throws Exception {
+        this.module.remove();
```
Comment (stuartwdouglas) [inline testsuite/integration/secman/src/test/java/org/jboss/as/testsuite/integration/secman/custompermissions/AbstractCustomPermissionServerSetup.java]: This looks like it does not reverse the minimum permissions changes that it makes

Comment (hsvabek) [inline testsuite/integration/secman/src/test/java/org/jboss/as/testsuite/integration/secman/custompermissions/AbstractCustomPermissionServerSetup.java]: I added backup/restore minimum permissions. It is ok now?

**Comment by hsvabek** [comment]
retest this please

**Comment by kabir** [comment]
Retest this please

**Comment by kabir** [comment]
@stuartwdouglas please approve

**Comment by stuartwdouglas** [comment]
I think that maybe the code in Utils should just be in the base test class. Otherwise it looks ok.

**Pull request merged**